spec/features/bus_delete_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'deleting a bus - admin' do 
```

Comment:
I might suggest structuring this test differently to remove some of the duplication:

```ruby
describe 'deleting a bus'
  before(:each) do
    some_setup
  end
  context 'as admin' do
    it 'works'
  end
  # etc.
end
```